packages/web/src/common/store/account/sagas.js

```diff
@@ -240,12 +240,12 @@ export function* fetchLocalAccountAsync() {
   const cachedAccountUser = yield call([localStorage, 'getAudiusAccountUser'])
   const currentUserExists = yield call([localStorage, 'getCurrentUserExists'])
   if (cachedAccount && cachedAccountUser && !cachedAccountUser.is_deactivated) {
+    yield put(fetchAccountSucceeded(cachedAccount))
```

Comment:
Could this be dangerous? Something listening to account succeeded might depend on it being in the cache

Reply:
I think I can drop this line now actually. Testing